File: RemoveDuplicatesFromSortedArray/removeDuplicatesFromSortedArray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {

  let modifiedArr = [];
  for (let i = 0; i < nums.length; i++) {
    if (!modifiedArr.includes(nums[i])) {
      modifiedArr.push(nums[i]);
    }
  }
  nums.length = 0;
  nums.push(...modifiedArr);
  return modifiedArr.length;
};

const data = [1,1,2,3,3,4];

console.log(removeDuplicates(data));
console.log(data);

File: RemoveDuplicatesFromSortedArray/removeDuplicatesFromSortedArray.py
class Solution:
    def removeDuplicates(self, nums):

        leftIndex = 0
        for items in nums:
            if items != nums[leftIndex]:
                leftIndex += 1
                nums[leftIndex] = items
        return leftIndex + 1
    

nums = [1,1,2,2,3,3,4,4,5]
sol = Solution()
print(sol.removeDuplicates(nums))
print(nums)